src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -86,7 +86,13 @@
                 "OperationTimeout": "00:00:30"
             },
             "Validate": {
-                "CacheDurationInSeconds": 14400
+                "CacheDurationInSeconds": 20000,
+                //"ExternalTerminologyServer": "https://localhost:44349/"
```

Comment:
I don't think we want to check in these changes as default settings.  I think as a default, the ExternalTeminologyServer should not be set.  Once you change this, you will likely need to update the setting in your tests to make sure they work.

Reply:
The tests use testConfiguration.json settings I believe, so this file should not affect them. Would it be fine to keep the testConfiguration.json changes?